tests/visualization_tests/matplotlib_tests/test_parallel_coordinate.py

```diff
@@ -92,7 +154,19 @@ def test_plot_parallel_coordinate_categorical_params() -> None:
         )
     )
     figure = plot_parallel_coordinate(study_categorical_params)
-    assert len(figure.get_lines()) == 0
+    axes = figure.get_figure().axes
+    assert len(axes) == 3 + 1
+    assert axes[0].get_ylim() == (0.0, 2.0)
+    assert axes[1].get_ylabel() == "Objective Value"
+    assert axes[1].get_ylim() == (0.0, 2.0)
+    assert axes[2].get_ylim() == (0, 1)
+    assert [lien.get_text() for lien in axes[2].get_yticklabels()] == ["preferred", "opt"]
```

Comment:
A nit.
```suggestion
    assert [line.get_text() for line in axes[2].get_yticklabels()] == ["preferred", "opt"]
```

There are some this type of typos.